book/about_the_tutorial/Requirements.md

```diff
@@ -4,27 +4,33 @@
 The following are the requirements needed for the course:
 
 
-### Desirable:
+### Required:
 
-- Have completed the NASA Open Science 101 course.
+- **A free NASA EarthData user account.**
 
-- Have basic programming knowledge.
+  - To access the data used in this course, you will need a free NASA EarthData user account. You can create an account on the NASA EarthData website ([https://urs.earthdata.nasa.gov/ ](https://urs.earthdata.nasa.gov/)).
 
+* **A free user account on GitHub.**
 
-### Required:
+     - To download the course source code, you will need a free user account on GitHub. You can create an account on the GitHub website (<https://github.com/>).
 
-- A free NASA EarthData user account:
+- **A 2i2c hub user account.**
 
-  - To access the data used in this course, you will need a free NASA EarthData user account. You can create an account on the NASA EarthData website ([https://urs.earthdata.nasa.gov/ ](https://urs.earthdata.nasa.gov/)).
+  - To run the course notebooks, you will need a free 2i2c hub account. You can create an account on the 2i2c website (Link).
 
-* A free user account on GitHub:
+* **Fundamentals of Spatial Data.** 
+This course assumes a basic understanding of how geographic information is represented digitally. This includes concepts like points, lines, and areas used to represent locations on Earth.  You also should have knowledge about geospatial concepts such as coordinate systems and data formats.
```

Comment:
.. such as coordinate systems and **geographic** data formats.

Reply:
suggestion added